docker-compose.prod.yml

```diff
@@ -1,23 +1,31 @@
 version: '3.7'
 
 services:
-   db:
-     image: mariadb:10.3
-     volumes:
-       - db_data:/var/lib/mysql
-     restart: always
-     env_file: ./docker/.env.mysql
+  db:
+    image: mariadb:10.3
+    volumes:
+      - db_data:/var/lib/mysql
+    restart: always
+    env_file: ./docker/.env.mysql
+
+  lh-ehr:
+    image: registry.gitlab.com/librehealth/ehr/lh-ehr:latest
+    volumes:
+      - sites:/var/www/html/sites
+    ports:
+      - "8000:80"
+    restart: always
+    env_file: ./docker/.env.ehr
+    depends_on:
+      - db
+  adminer:
+    image: adminer:4
+    restart: always
+    ports:
+      - "8080:8080"
```

Comment:
8080 is typically the port Java TomCat runs under (think Jenkins) maybe a 8010 or something?

Reply:
what about 8001?

Reply:
Perfect :)